dev-site-config/site.config.js

```diff
@@ -6,4 +6,5 @@ module.exports = {
   sideEffectImports: [
     './initalizeXFC.js',
   ],
+  includeTestEvidence: false,
```

Comment:
Is this something we should set in the other monorepos?

Reply:
Yes, we don't need to include test evidence in open source and it just bloats the dev-site pages